dask_bigquery/tests/test_core.py

```diff
@@ -140,9 +177,7 @@ def write_dataset():
         (None, {"write_disposition": "WRITE_APPEND"}),  # non-default
         (
             None,
-            {
-                "source_format": bigquery.SourceFormat.AVRO
-            },  # non-default, should enforce PARQUET
+            {"source_format": bigquery.SourceFormat.AVRO},  # non-default, should enforce PARQUET
```

Comment:
I must be on a different version of `black` or something bc these appeared when I tried to lint locally ☹️

Reply:
If you do `pip install pre-commit` and `pre-commit install`, that should give you the same `black` as this repo uses.